web/src/drafts.js

```diff
@@ -241,6 +241,9 @@ function draft_notify() {
         placement: "right",
     })[0];
     instance.show();
+    if (instance.popper) {
+        instance.popper.querySelector(".tippy-content").style.whiteSpace = "nowrap";
+    }
```

Comment:
What's the story with this change? It seems like it might deserve its own commit message.

Reply:
The reason for this change is that in the new layout, the `saved as draft` tooltip was breaking into multiple lines instead of a single horizontal line when a draft was saved. I will create a separate commit to explain the reason for this change and add comments to explain the change as well. 
Before:
![image](https://user-images.githubusercontent.com/64723994/228286923-427324ab-0e43-4daa-ab1b-ce951a9ed5d0.png)
After:
![image](https://user-images.githubusercontent.com/64723994/228286994-cfa33943-7338-4c23-a46a-b8adf21b6596.png)